rentals.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'person'
+require_relative 'book'
+
+class Rentals
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] Please Getters and Setters for @rentals ,@age , name should be present. in Rentals class
![image](https://user-images.githubusercontent.com/74744787/219880803-de734670-12b3-47f5-a8d2-fc61fbdce0df.png)

Reply:
But it is not reuqired as per project requirment rest I have added all requested.
![Capture](https://user-images.githubusercontent.com/109624418/219882809-f32c77a5-6c28-4838-b0f8-32be6f332c86.JPG)